content/docs/user-guide/data-management/remote-storage/amazon-s3.md

```diff
@@ -0,0 +1,244 @@
+# Amazon S3
+
+<!--
+## Amazon S3
+-->
+
+Start with `dvc remote add` to define the remote. Set a name and valid [S3] URL:
+
+```cli
+$ dvc remote add -d myremote s3://<bucket>/<key>
+```
+
+- `<bucket>` - name of an [existing S3 bucket]
+- `<key>` - optional path to a [folder key] in your bucket
+
+Upon `dvc push` (or when needed) DVC will try to authenticate using your [AWS
+CLI config]. This reads the default AWS credentials file (if available) or
+[env vars](#environment-variables).
+
+[aws cli config]:
+  https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html
+
+<admon type="info">
+
+The AWS user needs the following permissions: `s3:ListBucket`, `s3:GetObject`,
+`s3:PutObject`, `s3:DeleteObject`.
+
+</admon>
+
+[s3]: https://aws.amazon.com/s3/
+[existing s3 bucket]:
+  https://docs.aws.amazon.com/AmazonS3/latest/gsg/CreatingABucket.html
+[folder key]:
+  https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-folders.html
+
+## Custom authentication
+
+If you don't have a regular local S3 setup or if you want to change the auth
```

Comment:
Not sure what `a regular local S3 setup` means, and `local` is used again with a different meaning at the end of the sentence. What about `If you don't have the AWS CLI installed/configured` or something to that effect if that's what you meant here?

Reply:
Going with this but I think AWS CLI is not the only way to setup a local S3 connection? Other apps or libraries may set the env vars as well. I guess it covers most people though:
```suggestion
If you don't have the AWS CLI configured in your machine or if you want to change the auth
```